modules/i18n/src/constants.ts

```diff
@@ -46,29 +46,13 @@ export class I18nModuleConstants {
      */
     public static readonly COMMON_NAMESPACE: string = "common";
 
-    /**
-     * Dev portal namespace.
-     * @constant
-     * @type {string}
-     * @default
-     */
-    public static readonly DEV_PORTAL_NAMESPACE: string = "devPortal";
-
-    /**
-     * Admin portal namespace.
-     * @constant
-     * @type {string}
-     * @default
-     */
-    public static readonly ADMIN_PORTAL_NAMESPACE: string = "adminPortal";
-
     /**
      * User portal namespace.
      * @constant
      * @type {string}
      * @default
      */
-    public static readonly USER_PORTAL_NAMESPACE: string = "userPortal";
+    public static readonly MYACCOUNT_NAMESPACE: string = "myAccount";
```

Comment:
Shall we rename this to `MY_ACCOUNT_NAMESPACE`. WDTY?

Reply:
+1 and fixed with commit 76d6c86